apa102/softspi.go

```diff
@@ -0,0 +1,68 @@
+package apa102
+
+import "github.com/tinygo-org/tinygo/src/machine"
```

Comment:
shouldn't this just be "machine" ?

Reply:
yeah it should.  I thought I had fixed that up before I committed but I think maybe goimports snuck it back in on me.  Good catch, thanks